docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2139,12 +2139,16 @@ BESU_SYNC_MODE=FAST
 sync-mode="FAST"
 ```
 
-The synchronization mode. The options are `FAST` and `FULL`. The default is `FULL`.
+The synchronization mode. The options are `FAST` and `FULL`.
+
+* The default is `FULL` when not using the [`--network`](#network) option.
```

Comment:
Technically you can get a FAST sync even when not using the network option. If you don't supply anything (genesis or network) then it's mainnet, so it's fast.

Reply:
@RatanRSur - so the default is FULL if you don't name a network and specify a genesis file?

Reply:
Yes

Reply:
Updated